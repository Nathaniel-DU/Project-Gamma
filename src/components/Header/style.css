#main-header {
    font-size: 2.3rem;
    color: #ADEFD1FF;
    text-align: center;
    padding-right: 15px;
}
