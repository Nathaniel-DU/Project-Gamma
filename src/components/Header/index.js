import React from 'react';
import './style.css'

export default function Header({text}) {
    return (
        <h1 id='main-header' className='header'>{text}</h1>
    );
};