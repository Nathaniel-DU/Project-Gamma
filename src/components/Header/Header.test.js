import React from 'react';
import { shallow } from 'enzyme';
import Header from './index';

describe(`Header`, () => {
    it(`should render correctly in "debug" mode`, () => {
        const component = shallow(<Header debug />);
        expect(component).toMatchSnapshot();
    });
});