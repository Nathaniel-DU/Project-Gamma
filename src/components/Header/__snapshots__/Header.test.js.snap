// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly in "debug" mode 1`] = `
<h1
  className="header"
  id="main-header"
/>
`;
